lib/plugins/aws/lib/naming.js

```diff
@@ -61,6 +65,10 @@ module.exports = {
     return 'serverless-state.json';
   },
 
+  getCloudFormationFileNameRegex() {
+    return /.*cloudformation-.+\.json/;
```

Comment:
This will unconditionally include CF files created by plugins that are now owned by Serverless. They should not be selected here.